py/trtorch/csrc/trtorch_py.cpp

```diff
@@ -9,12 +9,95 @@
 #include "torch/custom_class.h"
 #include "torch/script.h"
 #include "torch/torch.h"
+#include "util.h"
 
+using namespace nvinfer1;
+using namespace pybind11::literals;
```

Comment:
Just a style thing, I prefer to not use `using` since it masks namespaces later, if theres something we are using from TensorRT or pybind its easier if its clear where that object comes from.

Reply:
so for literals it would be better to do `py::literals::OBJECT`

Reply:
Changed it.